### DISCUSSION TITLE
完成“Solidity入门”如何获得SBT认证？

### DISCUSSION THREAD

**Discussion by linhaowei1** [AmazingAng/WTF-Solidity#277] [Q&A] [answered]
我完成了Solidity 入门教程的所有练习，但这个“毕业”button似乎是不能点击的？想知道这个是不是正常的呀？以及SBT认证如何获得呢？
感谢！
<img width="1171" alt="image" src="https://user-images.githubusercontent.com/56622542/196115725-083f57e7-fa73-46c8-8149-3552060dac1b.png">

**Comment by doraemonkeys** [comment, marked as answer]
等链上认证系统公测上线就可以领取了

**Reply by imimsuu** [reply]
我有很多80分，可以认证吗
